templates/taiga-front-configmap.yaml

```diff
@@ -32,6 +32,6 @@ data:
         "enableGithubImporter": ${ENABLE_GITHUB_IMPORTER},
         "enableJiraImporter": ${ENABLE_JIRA_IMPORTER},
         "enableTrelloImporter": ${ENABLE_TRELLO_IMPORTER},
-        "gravatar": false,
+        "gravatar": ${ENABLE_GRAVATAR},
```

Comment:
Question: Is this type of environment var "truthy" aka, does it evaluate to false if the value is not provided?